MantidQt/CustomInterfaces/src/EnggDiffraction/EnggDiffFittingPresenter.cpp

```diff
@@ -1467,7 +1470,7 @@ void EnggDiffFittingPresenter::setDefaultBank(
 
 bool EnggDiffFittingPresenter::isDigit(std::string text) const {
   // bool isDig = true;
-  for (size_t i = 0; i < text.size(); i++) {
+  for (auto i : text) {
     char *str = &text[i];
```

Comment:
Yes, I'd say that you'd use `i` which is the character directly as the parameter to `isdigit()`.